app/src/components/ExperimentDialog/ExperimentDialog.View.tsx

```diff
@@ -65,19 +63,13 @@ const ExperimentDialogView = ({
       </IonItem>
       <IonItem>
         <IonLabel position="fixed">File Format:</IonLabel>
-        <IonSelect
-          value={experimentFileFormat}
-          multiple={false}
-          onIonChange={changeExperimentFileFormat}
-        >
-          {$enum(experimentFileFormatEnum).map(
-            (form: SetExperimentFileFormatEnum) => (
-              <IonSelectOption key={'format-option-' + form} value={form}>
-                {form}
-              </IonSelectOption>
-            )
```

Comment:
Use Object.keys(experimentFileFormatEnum).map(...) instead because it is a default built-in.